### PR TITLE
themes graph component

### PR DESCRIPTION
**Pull request by conundrumgirl**
Themes plot component for CSBC
image

### CODE REVIEW

File: src/lib/containers/widgets/themes-plot/BarPlot.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  style?: React.CSSProperties
+  plotData: GraphItem[]
+  layoutConfig: Partial<PlotlyTyped.Layout>
+  optionsConfig: Partial<PlotlyTyped.Config>
+  label: string
+  xMax?: number
+  onClick?: Function
+}
+
+function getBarPlotDataPoints(data: any[], isFade = false, filter?: string) {
```
Comment (conundrumgirl) [inline src/lib/containers/widgets/themes-plot/BarPlot.tsx]: add return type

File: src/lib/containers/widgets/themes-plot/DotPlot.tsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+import _ from 'lodash-es'
+const Plot = createPlotlyComponent(Plotly)
+
+export type DotPlotProps = {
+  plotData: GraphItem[]
+  layoutConfig: Partial<PlotlyTyped.Layout>
+  optionsConfig: Partial<PlotlyTyped.Config>
+  label?: string, 
+  id: string,
+  isHeader?: boolean,
```
Comment (conundrumgirl) [inline src/lib/containers/widgets/themes-plot/DotPlot.tsx]: isHeader isFooter => isLegend isXaxis

File: src/lib/containers/widgets/themes-plot/DotPlot.tsx [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    'star-square-open',
+    'diamond-x',
+  ]
+
+  groups.forEach((group, i) => {
+    data.push({
+      type: 'scatter',
+
+      x: isFakeData
+        ? [-10]
```
Comment (conundrumgirl) [inline src/lib/containers/widgets/themes-plot/DotPlot.tsx]: add a note

File: src/lib/containers/widgets/themes-plot/DotPlot.tsx [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+function getPlotDataPoints(
+  graphItems: GraphItem[],
+  ySorted?: string[],
+): any[] {
+  const isFakeData = ySorted === undefined
+  var groups = _.uniq(graphItems.map(item => item.group))
+  let data: any = []
+  let symbols = [
+    'circle',
+    'triangle-up',
```
Comment (leem42) [inline src/lib/containers/widgets/themes-plot/DotPlot.tsx]: Note to self - look into options here

**Pull request merged**